.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules/
 /.idea/
 /dist/
+/e2e/
```

Comment:
Any special reason for this line?

Reply:
Yes. It's an output folder created when using CodeceptJS/Puppeteer with Mochaawsome. 
Forgot to include it in my earlier Puppeeter PR.